pipeline/utils/notifications_test.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from pipeline.types import NotificationInfo, NotificationType
+from pipeline.utils.notifications import notifications_per_user
+
+
+def test_return_lst_1_notification():
+    notifications = [
+        NotificationInfo(
+            recipient="A",
+            notification_type=NotificationType.CHECK_MATCH,
+            messagers="B",
+        )
+        # NotificationInfo(recipient="B", notification_type=NotificationType.CHECK_MATCH, messagers="A")
+    ]
+    actual = notifications_per_user(notifications)
+    expected = [
+        NotificationInfo(
+            recipient="A",
+            notification_type=NotificationType.CHECK_MATCH,
+            messagers="B",
+        )
+        # NotificationInfo(recipient="B", notification_type=NotificationType.CHECK_MATCH, messagers="A")
+    ]
+
+    assert actual == expected
+
+
+# def test_return_lst_with_double_notifications():
```

Comment:
Excellent, this is a good test case that tests for excluding repeat notifications!